vita3k/kernel/include/kernel/state.h

```diff
@@ -294,6 +305,9 @@ struct KernelState {
     SceKernelModuleInfoPtrs loaded_modules;
     LoadedSysmodules loaded_sysmodules;
     ExportNids export_nids;
+
+    InitialFibers initial_fibers;
```

Comment:
Fibers are stored on vita at the thread level. is it possible to move this to thread state?